sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -300,16 +300,19 @@ export const enum KnownDeletionRecoveryLevel {
 }
 
 // @public
-export const enum KnownEncryptionAlgorithms {
+export enum KnownEncryptionAlgorithms {
     A128CBC = "A128CBC",
+    A128CbcHS256 = "A128CBC-HS256",
```

Comment:
I think these were added to Java to keep parity with the Track 1 SDK. If JS did not offer these in Track 1 I don't think we need to add them right now.

Reply:
Ooh ok ok! Thank you!